src/pages/tasks/kanban/Kanban.tsx

```diff
@@ -237,11 +249,14 @@ export function Kanban() {
   useEffect(() => {
     projectId
       ? setApiEndpoint(
-          route('/api/v1/tasks?project_tasks=:projectId&limit=1000', {
-            projectId,
-          })
+          route(
+            '/api/v1/tasks?project_tasks=:projectId&limit=1000&per_page=500',
+            {
+              projectId,
+            }
+          )
         )
-      : setApiEndpoint('/api/v1/tasks?limit=1000');
+      : setApiEndpoint('/api/v1/tasks?limit=1000&per_page=500');
```

Comment:
May I know why this was added? We don't have pagination on the kanban page, thus limit & per_page make no sense.

Reply:
Sure, I have an explanation for the `per_page` meta data, but the `limit` was already there so I don't know what impact it has. So even if we don't have any pagination the API will return paginable data, by default I think it will be 15/20 results per page and when we fetch the data from the API only the first page will be set to the state that will be used to display data on the UI. If I exclude meta data `per_page` I always get 20 results even if the total result is 121 in my case. I've been researching about react queries and I think it's not about the settings in the react query options, it's about how our logic works on the API. Do you know of any way to remove the pagable logic in the sepcific request?

For better understanding here is the screenshot:

![Screenshot 2023-01-16 at 14 46 36](https://user-images.githubusercontent.com/51542191/212697473-c20cef79-b659-4474-be1c-e131363d9df3.png)

Here is also GenericManyResponse interface which includes this part for meta data:

```
export interface GenericManyResponse<T> {
  data: T[];
  meta: {
    pagination: {
      total: number;
      count: number;
      per_page: number;
      current_page: number;
      total_pages: number;
      links: string[];
    };
  };
}
```

Now, I just set `per_page` query parameter same as limit to 1000. Let me know your thoughts.